src/iexamine.cpp

```diff
@@ -4430,8 +4430,8 @@ static void smoker_activate( player &p, const tripoint &examp )
                      false ) );
             add_msg( _( "You remove %s from the rack." ), it.tname() );
             g->m.add_item_or_charges( p.pos(), it );
-            g->m.i_rem( examp, &it );
             p.mod_moves( -p.item_handling_cost( it ) );
+            g->m.i_rem(examp, &it);
```

Comment:
```suggestion
            g->m.i_rem( examp, &it );
```

Reply:
This formatting thing is the only thing holding up a merge.